lib/fizzy/execute.cpp

```diff
@@ -1470,6 +1470,16 @@ ExecutionResult execute(Instance& instance, FuncIdx func_idx, span<const Value>
             binary_op(stack, std::divides<float>{});
             break;
         }
+        case Instr::f32_copysign:
+        {
+            // TODO: This is not optimal implementation. The std::copysign() is inlined, but
```

Comment:
Why not implement it then? Is that planned after merging this? Did you only see this for `copysign`?

Reply:
Only for `copysign` and `abs`. Also theoretical UB will be involved, so I prefer to have all spec tests passed before doing that.